templates/project_view/index.html.twig

```diff
@@ -0,0 +1,56 @@
+{% extends 'base.html.twig' %}
+{% import "macros/widgets.html.twig" as widgets %}
+{% import "macros/datatables.html.twig" as tables %}
+{% import "macros/toolbar.html.twig" as toolbar %}
+{% import "activity/actions.html.twig" as actions %}
+
+{% set columns = {
+    'name': {'class': 'alwaysVisible', 'title': 'label.name'|trans},
+    'today': {'class': 'alwaysVisible', 'title': 'label.today'|trans},
+    'week': {'class': 'alwaysVisible', 'title': 'label.week'|trans},
+    'total': {'class': 'alwaysVisible', 'title': 'label.total'|trans},
+    'expected_duration': {'class': 'alwaysVisible', 'title': 'label.expected_duration'|trans},
+    'expected_delivery': {'class': 'alwaysVisible', 'title': 'label.expected_delivery'|trans},
+    'description': {'class': 'hidden-xs', 'title': 'label.description'|trans}
+} %}
+{% for field in metaColumns %}
+    {% set columns = columns|merge({
+        ('mf_' ~ field.name): {'title': field.label|trans, 'class': 'hidden-xs hidden-sm', 'orderBy': false}
+    }) %}
+{% endfor %}
+{% set columns = columns|merge({
+    'visible': {'class': 'text-center hidden w-min', 'orderBy': false},
+    'actions': {'class': 'actions alwaysVisible'},
+}) %}
+
+{% set tableName = 'project_view' %}
+
+{% block page_title %}{{ 'project_view.title'|trans }}{% endblock %}
+
+{% block main_before %}
+    {{ tables.data_table_column_modal(tableName, columns) }}
+{% endblock %}
+
+{% block main %}
+
+    {% if entries is null %}
+        {{ widgets.nothing_found('kimai.activityUpdate') }}
```

Comment:
the event is wrong. if you only have read logic, then this can be completely removed (same below in the table header)